LLMListView/SwipeReleaseAnimationConstructor.cs

```diff
@@ -182,7 +182,6 @@ public void SwipeTo(SwipeDirection direction, SwipeConfig config, bool animated)
             else
             {
                 config.MainTransform.X = targetX;
```

Comment:
Have you any ideas why it's equal to 0 even when it's defined OnLoad, etc? You can try OnLoad set _mainTransform to any value but it's reseted to 0 anyway. :bow: